notices.json

```diff
@@ -74,18 +74,18 @@
       "clientType": "all",
       "serverVersion": ["<10.2"],
       "instanceType": "onprem",
-      "displayDate": ">= 2024-11-18T00T00:00:00Z"
+      "displayDate": ">= 2024-11-18T00:00:00Z"
     },
     "localizedMessages": {
       "en": {
         "title": "Mattermost 10.2 is here!",
-        "description": "Mattermost v10.2 includes multiple new improvements and bug fixes, including [Connected Workspaces (Beta)](https://docs.mattermost.com/onboard/connected-workspaces.html). [Upgrading](https://docs.mattermost.com/upgrade/upgrading-mattermost-server.html) only takes a few minutes.",
```

Comment:
This change looks unintended (in the documentation still shows as beta). Is it correct?

Reply:
@larkox 
I think its intended. I can see beta is removed from the PR changes going into release PR
https://github.com/mattermost/notices/pull/417/